test_email_validation.py

```diff
@@ -0,0 +1,8 @@
+import email_validation
+import unittest
+
+
+class EmailValidationTest(unittest.TestCase):
+    """unit test for EmailValidation"""
```

Comment:
Docstrings are usually sentences (i.e. start with uppercase letter and end with a period)